### PR TITLE
Add launch-ovn option into cluster-bot for metal-ipi clusters

### PR DESCRIPTION
**Pull request by ardaguclu**
launch-ovn will create IPv6, OVNKubernetes metal-ipi cluster.

### CODE REVIEW

File: ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml [hunk 1]
```diff
@@ -1239,27 +1239,15 @@ periodics:
           - as: e2e
             steps:
               cluster_profile: packet
-              env:
-                DEVSCRIPTS_CONFIG: |
-                  IP_STACK=v4
-                  NETWORK_TYPE=OpenShiftSDN
-              workflow: baremetalds-e2e
+              workflow: baremetalds-e2e-ipv4
           - as: e2e-dualstack
```
Comment (vrutkovs) [inline ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml:1243]: ci-chat-bot doesn't have `dualstack` support iiuc, lets remove this test

File: ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml [hunk 2]
```diff
@@ -1239,27 +1239,15 @@ periodics:
...
             steps:
               cluster_profile: packet
-              env:
-                DEVSCRIPTS_CONFIG: |
-                  IP_STACK=v4v6
-                  NETWORK_TYPE=OVNKubernetes
-              workflow: baremetalds-e2e
+              workflow: baremetalds-e2e-ovn-dualstack
           - as: e2e-ipv6
```
Comment (vrutkovs) [inline ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml:1247]: This change needs to be moved to a different PR

File: ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml [hunk 3]
```diff
@@ -1239,27 +1239,15 @@ periodics:
           - as: e2e
```
Comment (vrutkovs) [inline ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml:1239]: Needs to be moved to a different PR to isolate the change

**Review by vrutkovs — CHANGES_REQUESTED** [review]
launch change lgtm, but the PR needs to be cleaned up

File: ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml (hunk 1 above)
Comment (ardaguclu) [reply ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml:1243]: From metal ipi perspective, dualstack is one of important environment to test. Why do you think ci-chat-bot does not support it?

As far as I know, `test e2e-dualstack 4.10 metal` works.

File: ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml (hunk 2 above)
Comment (ardaguclu) [reply ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml:1247]: They are exactly same workflows, latter one is newest, I can open a different PR.

File: ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml (hunk 1 above)
Comment (vrutkovs) [reply ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml:1243]: Ah, okay, I wasn't aware. In any case, it should be a different PR to isolate the change

**Comment by ardaguclu** [comment]
> launch change lgtm, but the PR needs to be cleaned up

@vrutkovs I just left this change and will open a different PR for others. Could you PTAL again?

**Review by vrutkovs — APPROVED** [review]
/lgtm
